### PR TITLE
fix(modal): medium screen and xs size

### PR DESCRIPTION
**Pull request by theJohnnyMe**
**Describe pull-request**  
Fixing value for the extra small size of modal on medium screen size.

**Solving issue**  
Fixes: [CDEP-3095](https://tegel.atlassian.net/browse/CDEP-3095)

**How to test**  
1. Check if the new value extra small size of the modal on medium screen is 50%.

[CDEP-3095]: https://tegel.atlassian.net/browse/CDEP-3095?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/core/src/components/modal/modal.scss [hunk 1]
```diff
@@ -20,7 +20,7 @@ $modals: (
     lg: 100%,
   ),
   $screen-md: (
-    xs: 75%,
+    xs: 50%,
```
Comment (mJarsater) [inline packages/core/src/components/modal/modal.scss:23]: Maybe not breaking but should be highlighted in the release. People may have content in the modal of a certain width and expect it to be 75% on md screens.

Comment (timrombergjakobsson) [reply packages/core/src/components/modal/modal.scss:23]: Good input!

**Review by timrombergjakobsson — APPROVED** [review]
👍

File: packages/core/src/components/modal/modal.scss (hunk 1 above)
Comment (theJohnnyMe) [reply packages/core/src/components/modal/modal.scss:23]: Yep, it came as user reported issue so we will write about it in Announcement channel to give users heads-up.

**Review by nathalielindqvist — APPROVED** [review]
Nice! 👌

**Pull request merged**